docs/MO_DG/prepare_model/customize_model_optimizer/Customize_Model_Optimizer.md

```diff
@@ -2,6 +2,11 @@
 
 @sphinxdirective
 
+.. meta::
+   :description: Explore Model Optimizer extensibility mechanism, which enables 
+                 support of new operations and custom transformations to optimize 
+                 intermediate representation.
```

Comment:
Right, I guess we should leave out this article, as "it will be kept for some time".
```suggestion
```